src/Compilers/CSharp/Portable/Compilation/InitializerSemanticModel.cs

```diff
@@ -260,7 +260,9 @@ internal override bool TryGetSpeculativeSemanticModelForMethodBodyCore(SyntaxTre
 
         protected override BoundNode RewriteNullableBoundNodesWithSnapshots(BoundNode boundRoot, Binder binder, DiagnosticBag diagnostics, bool createSnapshots, out NullableWalker.SnapshotManager snapshotManager)
         {
-            return NullableWalker.AnalyzeAndRewrite(Compilation, MemberSymbol, boundRoot, binder, diagnostics, createSnapshots, out snapshotManager);
+            var symbol = boundRoot is BoundPropertyEqualsValue { Property: SourcePropertySymbol { BackingField: var backingField } } ? backingField : MemberSymbol;
```

Comment:
>var symbol = boundRoot is BoundPropertyEqualsValue { Property: SourcePropertySymbol { BackingField: var backingField } } ? backingField : MemberSymbol; [](start = 12, length = 151)

When ```MemberSymbol``` is a property we want to get its backing field instead? Is this what we are doing here?

Reply:
Correct, but this will be removed from the next iteration. We no longer need it as we're no longer assuming that the symbol `NullableWalker` is provided is the containing symbol.

---
In reply to: [327344300](https://github.com/dotnet/roslyn/pull/38769#discussion_r327344300) [](ancestors = 327344300)

Reply:
> Correct, but this will be removed from the next iteration.

Sounds good. Otherwise dependency on bound node feels unnecessary to me, we could jut figure this out from the ```MemberSymbol``` itself.

---
In reply to: [327347067](https://github.com/dotnet/roslyn/pull/38769#discussion_r327347067) [](ancestors = 327347067,327344300)